contracts/contracts/strategies/CompoundStrategy.sol

```diff
@@ -82,22 +83,24 @@ contract CompoundStrategy is BaseCompoundStrategy {
     }
 
     /**
-     * @dev Deposit the entire balance of any supported asset into Compound
+     * @notice Deposit the entire balance of any supported asset in the strategy into the underlying platform
      */
     function depositAll() external override onlyVault nonReentrant {
-        for (uint256 i = 0; i < assetsMapped.length; i++) {
-            uint256 balance = IERC20(assetsMapped[i]).balanceOf(address(this));
-            if (balance > 0) {
-                _deposit(assetsMapped[i], balance);
+        uint256 assetCount = assetsMapped.length;
+        for (uint256 i = 0; i < assetCount; ++i) {
+            IERC20 asset = IERC20(assetsMapped[i]);
+            uint256 assetBalance = asset.balanceOf(address(this));
```

Comment:
I like the `assetBalance` here, to distinguish from the later cTokenBalances. Usually it's better to just keep the shortest possible local name, rather than putting everything in it, but here it works well.